docs-api/documentation.md

```diff
@@ -0,0 +1,370 @@
+coremltools API v4.1 Documentation
```

Comment:
Please, let's not hard code version numbers.

Reply:
My bad, thanks for catching this.